src/signalmanager/os_signalmanager_windows.go

```diff
@@ -9,16 +9,7 @@ import (
 
 func RegisterKillSignalChannel() chan os.Signal {
 	killChan := make(chan os.Signal)
-	signal.Notify(killChan, os.Kill, os.Interrupt)
+	signal.Notify(killChan, os.Interrupt)
```

Comment:
I think, it does work on windows. We had separate files because of the other go-routine dump functionality.